tests/test.spec.ts

```diff
@@ -70,6 +71,17 @@ it(`Import regex matches import statements`, () => {
   });
 });
 
+it(`Import regex does not match edge cases from keywords in strings`, function () {
+  const testCase = `
+    'a string with keyword from '
+    // The from keyword in that string can cause
+    // a match up to the next quote, since the regex does not
+    // know that the keyword is in a string context
+    'another string using same quote type'
+  `;
+  expect(newImportStatementRegex().exec(testCase)?.[0]).toBeUndefined();
```

Comment:
Stripped down version of the scenario I ran into in production, where the word "for" inside a string caused a spurious multiline string to be treated as an import path. This test was failing until I updated the regexes.